Peihong's Project 1 Completed

peihongx opened the pull request:
@johnbeve I've been reviewed by @dmcnulty27 and @giacomodecolle and done necessary revisions.

peihongx committed:
Update README.md

peihongx committed:
Update README.md

peihongx committed:
TRY TO USE VS CODE

peihongx committed:
test code

peihongx committed:
assignment

peihongx committed:
assignment

peihongx committed:
Update README.md

peihongx committed:
ssss

peihongx committed:
Merge branch 'main' of https://github.com/peihongx/PHI-696

peihongx committed:
ssss

peihongx committed:
sssss

peihongx committed:
sssss

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
Fix image display

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
7-10

peihongx committed:
only 7

peihongx committed:
complete

peihongx committed:
complete

peihongx committed:
complete

peihongx committed:
Update image commits

peihongx committed:
delete proj1

peihongx committed:
Add proj1

peihongx committed:
Merge branch 'main' of https://github.com/peihongx/PHI-696

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

peihongx committed:
Project1 Completed (#2)

* Typo fix (#5)

The project was copy-and-pasted, but not all deets were updated to reflect the project is the 2nd one

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Create README.md

* Update README.md

Fixed question 8 based on Jieming's note.

* Update README.md

You can tag your peers in a commit @jonathanvajda @SydCo99 @giacomodecolle @oliviahobai @Jiemingy @JaronJCheung 

If you want someone to look over your work before you open a pull request, just tag them in the commit.

* Add proj1

* Delete README.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

* Update Peihong_Xie_Project_1_50375781.md

---------

Co-authored-by: Jonathan Vajda <36548200+jonathanvajda@users.noreply.github.com>
Co-authored-by: Jiemingy <123851348+Jiemingy@users.noreply.github.com>
Co-authored-by: John Beverley <johnbeve@buffalo.edu>

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

peihongx committed:
Fix image display issues

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

peihongx committed:
Merge branch 'main' of https://github.com/johnbeve/PHI-696 into johnbeve-main

peihongx committed:
Johnbeve main (#4)

* Typo fix (#5)

The project was copy-and-pasted, but not all deets were updated to reflect the project is the 2nd one

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Create README.md

* Update README.md

Fixed question 8 based on Jieming's note.

* Update README.md

You can tag your peers in a commit @jonathanvajda @SydCo99 @giacomodecolle @oliviahobai @Jiemingy @JaronJCheung 

If you want someone to look over your work before you open a pull request, just tag them in the commit.

* Add proj1

* Delete Peihong_Xie_Project_1_50375781.md

Hi there,
Merging to my repository is what we'll be doing next week. This week, I'd like you to open a pull request in your _own_ repository, then assign it to at least two of your peers from class. You'll all then review your updates, and once your reviewers give you the green light, you'll merge your changes to your branch, then open a pull request for me to review.
-John

* Delete fitch_proof_1.PNG

* Delete fitch_proof_2.PNG

* Delete fitch_proof_3.PNG

* Delete fitch_proof_4.PNG

* Delete tree_proof_1.png

* Delete tree_proof_2.png

* Delete tree_proof_3.png

* Delete tree_proof_4.png

* Create README.md

* Update README.md

* Update README.md

* Update README.md

---------

Co-authored-by: Jonathan Vajda <36548200+jonathanvajda@users.noreply.github.com>
Co-authored-by: Jiemingy <123851348+Jiemingy@users.noreply.github.com>
Co-authored-by: John Beverley <johnbeve@buffalo.edu>

peihongx committed:
Merge branch 'johnbeve-main'

peihongx committed:
Starting work

peihongx committed:
Q1,2,6 Completed

johnbeve commented on Project-1/Peihong_Xie_Project_1_50375781.md:
```diff
@@ -0,0 +1,299 @@
+
+(a) B(x,y) $\leftrightarrow$ ∃u(P(u,x)∧P(u,y))∧¬F(x)∧¬(x=y)
+
+(b) A(x,y) $\leftrightarrow$ ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x)
```
Counterexample: Aunts by marriage

peihongx replied:
> Counterexample: Aunts by marriage

@johnbeve Dear John, I think we should not cosider complexities induced by marriage. After all, we not only have "aunt by marriage, but also have "parent by marriage". Suppose that Jack's biological father died, and his biological mother is married to Duke so Duke becomes Jack's stepfather. So should the possibility of **being stepfather** be covered by P (parent_of)? Also, if we should consider aunts by marriage, then we should also consider brothers by marriage and cousins by marriage, leading to the result that B(x, y) and C (x, y) would also face similar difficulties. I don't think such complexities are necessary.

johnbeve replied:
@peihongx I gave you 'parent of' for free, so you're not on the hook for modeling step-parents. 
That said, if your argument is:

1. If we should model aunt by marriage then we should model brother/cousin by marriage
2. We should not model brother/cousin by marriage
3. Hence, we should not model aunt by marriage
 
Then rather than have me accept 3 as you intend, you'll force me to reject 2. Are you sure you want to offer this argument? Maybe a better idea would be to rescind the argument ;)

peihongx replied:
@johnbeve John, I don't think that "aunt by marriage" could be generally defined only by "parent_of". In order to express "aunt by marriage", a core thing is to express "x is a wife of y". Ok, if x and y has a child z, then we can do this by saying that x is a female prarent of z, y is a male parent of z, and x is not equal to y. But if x and y have no child, then it is impossible to express "x is a wife of y" only in terms of "parent_of" because y and y's wife have no relation by blood.

johnbeve commented on Project-1/Peihong_Xie_Project_1_50375781.md:
```diff
@@ -0,0 +1,299 @@
+
+(b) A(x,y) $\leftrightarrow$ ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x)
+
+(c) C(x,y) $\leftrightarrow$ ∃u∃w∃z(P(w,z)∧P(w,u)∧P(z,x)∧P(u,y)∧¬(z=u))∧¬(x=y)
```
Counterexample: Second cousins

peihongx replied:
> Second cousins

I revise my formula to adapt to the possibility of second cousins. But I guess that we can have third or fourth or ... cousin by appealing to great-great-grandparent or great-great-great-grandparent or ... In this way, such a formula can be constructed to be infinitely long. Just as Ali said, maybe all people are cousins on this giant piece of rock.

johnbeve commented on Project-1/Peihong_Xie_Project_1_50375781.md:
```diff
@@ -0,0 +1,299 @@
+
+(a)
+
+![picture 1](https://github.com/peihongx/PHI-696/blob/main/Project-1/tree_proof_1.png)
```
What justifies the move from a negated universal in line 1 to a negated universal in line 2?

peihongx replied:
> What justifies the move from a negated universal in line 1 to a negated universal in line 2?

If this move cannot be justified, it must be the case that the online tree proof generator goes wrong because this move is just what the generator does. Anyway, I read G. Priest's _An Introduction to Non-Classical Logic_, and use his tree-constructing rule for quantifers to rewirte these 4 tree proof manully. This must be what you want.

peihongx committed:
Q3, 9, 10 to be continued

peihongx committed:
unsure Q3, Q8 (useless?), Q10 (disjoint)

peihongx committed:
Q3, 8, 10 （disjoint)

peihongx committed:
typo fix

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

fix error

peihongx committed:
revise tree proof

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

fixing a problem with showing pictures

peihongx commented:
@johnbeve Hi John, I have revised my project 1 except for the issue of "aunts by marriage".

peihongx committed:
Q3, Q8 useless Q10 disjoint

peihongx committed:
Update Peihong_Xie_Project_1_50375781.md

revising Q4 and Q5

peihongx committed:
fix Q3, Q8, Q10

peihongx committed:
Q8, Q10

peihongx committed:
Q8, Q10

peihongx committed:
Fix owl file

peihongx committed:
fix typos, Q10 disjoint

peihongx committed:
Peihongkarl project 2 (#7)

* Update Peihong_Xie_Project_2_50375781.md

fix typo

* Update Peihong_Xie_Project_2_50375781.md

I'm unsure to how to infer a conclusion about D(a, e) and D(a, g)

* Update Peihong_Xie_Project_2_50375781.md

fix typo

peihongx committed:
Update Peihong_Xie_Project_2_50375781.md

title fixing

peihongx committed:
update proj-3 (#11)

* Update README.md (#53)

* Update README.md

* Update README.md

---------

Co-authored-by: John Beverley <johnbeve@buffalo.edu>

peihongx committed:
Q5, Q8, Q6-Unsure

peihongx committed:
Update Peihong_Xie_Project_2_50375781.md (#9)

* Update Peihong_Xie_Project_2_50375781.md

* empty

* Revert "empty"

This reverts commit f671cbe56485c13880f7c8875676afa7e169962b.

* Update Peihong_Xie_Project_2_50375781.md

* Q10 done

* Update Peihong_Xie_Project_2_50375781.md

* fix pic

* fix error

* Update Peihong_Xie_Project_2_50375781.md

Q10 is done now!

* Update Peihong_Xie_Project_2_50375781.md

* Update Peihong_Xie_Project_2_50375781.md

peihongx committed:
Merge branch 'main' of https://github.com/peihongx/PHI-696

peihongx committed:
Pull Request 4: Karl's kata challenges (#15)

* First draft of Karl's kata challenges

* Add more challenges, INSERT, IF function

* Update Peihong_Karl_Proj_4_50375781.md

SPARQL grammar highlight in Markdown. Thanks, Tim!

* Update Peihong_Karl_Proj_4_50375781.md

fix URI of dbc

* Update Peihong_Karl_Proj_4_50375781.md

fix a typo in challenge 9

* Update Peihong_Karl_Proj_4_50375781.md

mars case annotation

peihongx committed:
Add proj_5.file

peihongx committed:
Merge branch 'main' of https://github.com/peihongx/PHI-696

peihongx committed:
renamed

peihongx committed:
add BFO OWL version

peihongx committed:
Proj-5 Final submission

peihongx committed:
upload final project

peihongx committed:
move location#

peihongx committed:
Add two pdf files
